cmake/configure/configure_30_kokkos.cmake

```diff
@@ -17,4 +17,26 @@
 # Configuration for Kokkos support in deal.II:
 #
 
+SET(DEAL_II_WITH_KOKKOS ON # Always true. We need it :-]
+  CACHE BOOL "Build deal.II with support for Kokkos." FORCE
+  )
+
+MACRO(FEATURE_KOKKOS_CONFIGURE_BUNDLED)
+  LIST(APPEND KOKKOS_LIBRARIES ${CMAKE_DL_LIBS})
```

Comment:
Not quite sure if this is sufficient to make sure we link with `libdl`. If we can't guarantee that, we can't use any `Kokkos` profiling tools with the bundled installation which is probably not a big deal.

Reply:
Yeah, it doesn't matters.

Reply:
Yes, let's just disable it. That should also get the `MSVC` build going.